easybuild/easyconfigs/b/BAGEL/BAGEL-1.2.2-foss-2023a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'BAGEL'
+version = '1.2.2'
+
+homepage = "http://www.nubakery.org"
+
+description = """
+BAGEL (Brilliantly Advanced General Electronic-structure Library)
+is a parallel electronic-structure program.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/nubakery/bagel/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['%(name)s-%(version)s.patch']
+
+checksums = [
+    {'v1.2.2.tar.gz': 'b9ef9ad5ad9b836b5393badb5a3e5b162d6ff32da497f69d05ef9e4d091b558a'},
+    {'BAGEL-1.2.2.patch': '6a0bb2b0c762626219b9216cd96d287e7c14af72234e7ec40c5d684010747792'},
```

Comment:
checksum is still not correct.
simplest way to fix this:
```
eb BAGEL-1.2.2-foss-2023a.eb --inject-checksums --force
```

Reply:
When updating the PR after injecting the new checksum, it is deleting the main BAGEL.eb file.

Reply:
Had issues with pushing the updated files, but looks fine now.